packages/react/src/components/CollapsibleNavbarItem/collapsible-navbar-item.scss

```diff
@@ -0,0 +1,223 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+@mixin selectedItemBar {
+  content: "";
+  height: 80%;
+  width: 5px;
+  background-color: var(--oxygen-palette-primary-main);
+  position: absolute;
+  left: -8px;
+  border-radius: var(--oxygen-shape-borderRadius);
+}
+
+.oxygen-collapsible-navbar-item {
+  flex-shrink: 0;
+  white-space: nowrap;
+  box-sizing: border-box;
+  border-radius: var(--oxygen-shape-borderRadius);
+  transition: background-color 1s ease-out;
+  margin-top: 5px;
+  margin-bottom: 5px;
+
+  .oxygen-list {
```

Comment:
Seems like style blocks are differently indented from here onwards. Is that intentional? if it's not, let's fix indentation :)

Reply:
This is the formatting style of Oxygen.